### ISSUE TITLE
Basic authentication

### ISSUE THREAD

**Issue by sdavides** [lovelylain/hass_ingress#32]
what is the correct setting for backend basic authentication (popup login browser)?
I have tried various configurations it always replies 401 unauthorized

**Comment by lovelylain** [comment]
https://github.com/lovelylain/hass_ingress/issues/11#issuecomment-2227818632

**Comment by sdavides** [comment]
I had already tried but it doesn't work, probably it's my very old backend and made with absolute paths and redirects.
I'm sure of the base64 value I entered.

I created an add-on with nginx on "ingress" and it also creates an additional port listening with the authentication of homeassistant require.

I also implemented a base64 converter for basic authentication.

from ingress:
it works correctly, but I have problems with some scripts and redirects because the url contains "/hassio/ingress..."

from the additional port:
it works correctly because I directly call nginx on another port and the paths are clean.

https://github.com/sdavides/NginxReverse-Homeassistant

Thankyou
